include/kdbush.hpp

```diff
@@ -1,64 +1,89 @@
+// https://github.com/mourner/kdbush.hpp
+
 #pragma once
 
 #include <algorithm>
 #include <cmath>
 #include <vector>
 
+template<class TFloat = double, class TIndex = std::size_t>
```

Comment:
If TCoord doesn't sound too good, how about TNumber?

Thank you for your kind words, really happy to read that! Let me know how the KDBush library works out in your project.